cookbooks/bcpc/recipes/bootstrap_network.rb

```diff
@@ -73,6 +73,12 @@
   line "supersede routers #{router};"
 end
 
+replace_or_add 'append search domain' do
```

Comment:
Erm, am I missing the `Berksfile` which pulls in the [line-cookbook](https://github.com/someara/line-cookbook)?